client/src/tun2socks/Taskfile.yml

```diff
@@ -0,0 +1,111 @@
+version: '3'
+
+vars:
+  # This needs to be updated if the file moves.
+  REPO_ROOT: '{{joinPath .TASKFILE_DIR "../../.."}}'
+  # TODO: Output to $REPO_ROOT/output/client/tun2socks instead.
+  OUT_DIR: '{{joinPath .REPO_ROOT "client/output/build"}}'
+  BIN_DIR: "{{.OUT_DIR}}/build"
+  MOBILE_PKG: "{{.TASKFILE_DIR}}/outline/tun2socks"
+  GOMOBILE_BIND_CMD: "env PATH=\"{{.BIN_DIR}}:${PATH}\" '{{.BIN_DIR}}/gomobile' bind -ldflags='-s -w'"
+
+tasks:
+  electron:
+    desc: "Build the tun2socks binary for Electron platforms"
+    internal: true
+    requires: {vars: [TARGET_OS]}
+    vars:
+      # TODO(fortuna): remove this, it's not really needed since we don't release it separately anymore.
+      TUN2SOCKS_VERSION: "v1.16.11"
+      OUTPUT: '{{.OUT_DIR}}/{{.TARGET_OS}}/tun2socks{{if eq .TARGET_OS "windows"}}.exe{{end}}'
+    cmds:
+      - rm -rf "{{dir .OUTPUT}}" && mkdir -p "{{dir .OUTPUT}}"
+      # Linker flags: https://pkg.go.dev/cmd/link
+      # -s Omit the symbol table and debug information.
+      # -w Omit the DWARF symbol table.
+      # -X Set the value of the string variable.
+      - |
+        {{if ne OS .TARGET_OS -}}
```

Comment:
Ugh, it turns out OS is a function, not a variable, so it doesn't take the `.`. That was breaking the build.